sdk/compute/Azure.ResourceManager.Compute/src/Generated/VirtualMachineData.cs

```diff
@@ -235,19 +235,7 @@ public ResourceIdentifier HostGroupId
         /// <summary> Specifies the scale set logical fault domain into which the Virtual Machine will be created. By default, the Virtual Machine will by automatically assigned to a fault domain that best maintains balance across available fault domains.&lt;br&gt;&lt;li&gt;This is applicable only if the &apos;virtualMachineScaleSet&apos; property of this Virtual Machine is set.&lt;li&gt;The Virtual Machine Scale Set that is referenced, must have &apos;platformFaultDomainCount&apos; &amp;gt; 1.&lt;li&gt;This property cannot be updated once the Virtual Machine is created.&lt;li&gt;Fault domain assignment can be viewed in the Virtual Machine Instance View.&lt;br&gt;&lt;br&gt;Minimum api‐version: 2020‐12‐01. </summary>
         public int? PlatformFaultDomain { get; set; }
         /// <summary> Specifies Scheduled Event related configurations. </summary>
-        internal ScheduledEventsProfile ScheduledEventsProfile { get; set; }
-        /// <summary> Specifies Terminate Scheduled Event related configurations. </summary>
-        public TerminateNotificationProfile ScheduledEventsTerminateNotificationProfile
-        {
-            get => ScheduledEventsProfile is null ? default : ScheduledEventsProfile.TerminateNotificationProfile;
-            set
-            {
-                if (ScheduledEventsProfile is null)
-                    ScheduledEventsProfile = new ScheduledEventsProfile();
-                ScheduledEventsProfile.TerminateNotificationProfile = value;
-            }
-        }
```

Comment:
This is a breaking change.
We need to add this piece of removed code into a customized code, and add the attribute `EditorBrowsable` to hide it

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/33553/commits/d7b02899abbf4577807beedcf6e1a67e40433e71 
added customization here. could you check?